Misc/ACKS

```diff
@@ -1881,3 +1881,4 @@ Aleksandr Balezin
 Robert Leenders
 Tim Hopper
 Dan Lidral-Porter
+Ngalim Siregar
```

Comment:
Please note that we don’t append to this file, but keep it sorted alphabetically by author last name.